InsertionChangeLogGenerator/ChangeLogGenerator.cs

```diff
@@ -29,7 +29,8 @@ private static async Task<List<Commit>> GetCommitDetails(GitHubClient gitHubClie
             var processedCommits = new List<Commit>();
             foreach (var ghCommit in githubCommits)
             {
-                var commit = new Commit(ghCommit.Sha, ghCommit.Author.Login, $"https://github.com/{orgName}/{repoName}/commit/{ghCommit.Sha}", ghCommit.Commit.Message);
+                var author = ghCommit.Author?.Login ?? ghCommit.Committer?.Login ?? "";
```

Comment:
Out of curiosity, why would author be null? 
Deleted account?

Reply:
I believe this is what happens when someone uses git without doing the whole `git config --global user.name/user.email ...` dance.